Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -1073,7 +1074,7 @@ def list_endpoint_info(account_id, client_id, auth_token, ip, hostname, USE_SSL)
                         'MACAddress': assets['nics'][0]['mac_address'],
                         'OS': assets['os_info']['os_platform'],
                         'OSVersion': assets['os_info']['os_version'],
-                        'Model': assets['computer_info']['model'],
+                        'Model': str(assets.get('computer_info', {}).get('model')),
```

Comment:
in case we get None, str(None) returns a string None. Are you sure about it?
may I suggest:
assets.get('computer_info', {}).get('model'), '')

Reply:
@rshunim Returning string None is ok.